apollo-router/src/configuration/cors.rs

```diff
@@ -1,28 +1,141 @@
 //! Cross Origin Resource Sharing (CORS configuration)
+//!
+//! This module provides configuration structures for CORS (Cross-Origin Resource Sharing) settings.
+//!
+//! # Default Behavior
```

Comment:
Comments here won't show up in our JSON schema. But let's keep it for now, as our schema generation isn't that good with doc comments anyways. I might try to improve it a bit after https://github.com/apollographql/router/pull/7893.